mobify/astro-scaffold
# add deeplinking code to scaffold

I had added some code to AppDelegate in Sandbox to make deeplinking work but I didn't add it to the scaffold project. The user will still need to define their custom url scheme for deeplinking to work (unless Shawn puts that into his generator script.

Status: **Opened for visibility**

Reviewers: @jansepar @jeremywiebe @jvaill @MikeKlemarewski 
JIRA: **NONE**
Linked PRs: **https://github.com/mobify/astro/pull/183**
## Changes
- Copy and paste some boilerplate deeplinking code into the scaffold project.
## How to test-drive this PR
- bring up scaffold project in simulator
- enter a custom Url scheme under "URL types" (instructions: https://github.com/mobify/astro/blob/develop/guides/deeplinks.md)
- drop a break point in openURL in AppDelegate
- go to Safari in the simulator and enter your url. You should see the app open and your breakpoint hit.
## TODOS:
- [ ] Ensure changes to docs work well by changing them to Readme.io
